solar_farm.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import os
+from numpy.random import randint
+
+class SolarFarm:
+    def __init__(self):
+        self.dt = 0.5
+        self.efficiency=0.95
+        self.scenario = {}
+        self.bill = np.zeros(48) # prix de vente de l'électricité
+        self.load_battery = np.zeros(48) # chargement de la batterie (li)
+        self.battery_stock = np.zeros(49) #a(t)
+        self.capacity =100 #vérifier
+        self.max_load=70 #vérifier

```

Comment:
I suggest that these parameter could be defined when initializing the class e.g.  `__init__(self, capacity, max_load)`